src/graphic/Fast3D/gfx_sdl2.cpp

```diff
@@ -632,13 +630,15 @@ static inline void sync_framerate_with_timer() {
 #endif
     }
 
-#ifdef _WIN32
     t = qpc_to_100ns(SDL_GetPerformanceCounter());
     while (t < next) {
-        YieldProcessor(); // TODO: Find a way for other compilers, OSes and architectures
+#ifdef _WIN32
+        YieldProcessor();
+#else
+        sched_yield();
+#endif
         t = qpc_to_100ns(SDL_GetPerformanceCounter());
     }
-#endif
     t = qpc_to_100ns(SDL_GetPerformanceCounter());
```

Comment:
it seems strange to have this extra
```cpp
t = qpc_to_100ns(SDL_GetPerformanceCounter());
```
in here.

it seems it was always an odd "extra" call for windows, and it makes sense that it existed for other platforms because all of the other ones were inside the windows ifdef